node/src/chain_specs/calamari.rs

```diff
@@ -62,6 +63,8 @@ pub fn calamari_development_config() -> CalamariChainSpec {
                     get_account_id_from_seed::<sr25519::Public>("Alice"),
                     SessionKeys::new(get_collator_keys_from_seed("Alice")),
                 )],
+                // Delegations
+                vec![],
```

Comment:
can you add at least 1 delegator for easier local testing

Reply:
postponed to #798